src/OpenTelemetry.Api/Context/Propagation/BaggagePropagator.cs

```diff
@@ -138,7 +138,11 @@ internal static bool TryExtractBaggage(
                     break;
                 }
 
+#if NET
+                if (pair.IndexOf('=', StringComparison.Ordinal) < 0)
```

Comment:
I understand it is recommended to specify string comparison type. However, is there a way to configure a global default if the entire project always use `Ordinal`?

This feels contrary to C# tradition which gives a syntax sugar for many stuff.

Reply:
I see 3 options:
1. Keep as is now
2. Mute warning
3. Crete extensions methods - similar to object.OridnalIndexOf(charHEre)

I do not see any option to set the preferred method in other ways. Or it is not possible without writing custom analyzer.